docs/readthedocs/models/pv_model.md

```diff
@@ -202,11 +202,11 @@ $$
 Calculating the air mass ratio by dividing the radius of the earth with approx. effective height of the atmosphere (each in kilometer)
 
 $$
-airmassratio = (\frac{6371 km}{9 km}) = 707.8\overline{8}
+\airmassratio = (\frac{6371 km}{9 km}) = 707.8\overline{8}
 $$
 
 $$
-airmass = \sqrt{(707.8\overline{8} \cdot \cos({\theta_z}))^2 +2 \cdot 707.8\overline{8} +1)} - 707.8\overline{8} \cdot \cos{(\theta_z)})
+\airmass = \sqrt{(707.8\overline{8} \cdot \cos({\theta_z}))^2 +2 \cdot 707.8\overline{8} +1)} - 707.8\overline{8} \cdot \cos{(\theta_z)})
```

Comment:
This is not rendered well

Reply:
Can you specify what exactly the problem is?

Reply:
Sure, sorry. Next time I should directly add a screenshot. 

![grafik](https://github.com/ie3-institute/simona/assets/98817556/3133b009-388e-4d3b-86ff-c5fec403c46e)

see generated docu here: [https://simona--685.org.readthedocs.build/en/685/models/pv_model.html](https://simona--685.org.readthedocs.build/en/685/models/pv_model.html)